src/front/wgsl/mod.rs

```diff
@@ -265,8 +265,14 @@ impl Parser {
     fn get_shader_stage(word: &str) -> Result<crate::ShaderStage, Error<'_>> {
         match word {
             "vertex" => Ok(crate::ShaderStage::Vertex),
-            "fragment" => Ok(crate::ShaderStage::Fragment),
-            "compute" => Ok(crate::ShaderStage::Compute),
+            "fragment" => Ok(crate::ShaderStage::Fragment {
+                depth_replacing: None,
+                forced_early_depth_test: None,
+                conservative_depth_test: None,
+            }),
+            "compute" => Ok(crate::ShaderStage::Compute {
+                local_size: (8, 1, 1),
```

Comment:
same here, let's not pull out some non-trivial numbers

Reply:
Done